.github/workflows/pull-request.yml

```diff
@@ -8,6 +8,11 @@ on:
         required: false
         type: boolean
         default: true
+      devel:
```

Comment:
I think something got wrong during rebase/merge. Please make sure you're always rebasing your branches instead of merging develop into them